### ISSUE TITLE
bug while editing a quoted reply

### ISSUE THREAD

**Issue by voland66** [k9mail/k-9#4404]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
Tell us what should happen
If i switch to another app while writing inside quoted reply section of the email, changes to the quoted price should be saved in the draft and present when i return to k9 mail.
### Actual behavior
Tell us what happens instead
Any changes in the reply, wherever at the top or inside the quoted reply must be saved and available if/when i come back to k9 after opening another app.
### Steps to reproduce
1. Start a reply to a message. Clock in the pencil to edit quoted text. Make some edits 
2. Switch to any other appointments and then return to k9
3. All the changes in the quoted price of the reply are lost.

### Environment
K-9 Mail version: 5.700-5.703

Android version: 9

Account type (IMAP, POP3, WebDAV/Exchange): IMAP (gmail)

Please take some time to [retrieve logs](https://github.com/k9mail/k-9/wiki/LoggingErrors) and attach them here:

**Comment by voland66** [comment]
A little more information: it seems that k9 had problem saving a draft when switching to another app. I get a message "saved as a draft" but no draft exists. If i explicitly save the draft before switching to a different app, everything works as expected. The draft needs to be saved explicitly only once. As long as draft exists, it will be updated when necessary.

**Comment by cketti** [comment]
Works for me. Please create a new issue if this is still a problem with a recent beta version.
